### PR TITLE
update doc docker

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-provision-docker/pull/1
https://github.com/open-orchestra/open-orchestra-docs/pull/231

### CODE REVIEW

File: en/installation_guide/install_with_docker_for_contributors.rst [hunk 1]
```diff
@@ -11,176 +11,94 @@ Open Orchestra is composed of three projects:
...
-To install composer with curl:
+--------------
 
-.. code-block:: bash
+Install of Open Orchestra with docker require `Compose 1.6.0+` and `Docker Engine 1.10.+`.
 
-    $ curl -sS https://getcomposer.org/installer | php
+ - Install Docker for `Linux  <https://docs.docker.com/docker-for-mac/>`_
+ - Install Docker for `Mac  <https://docs.docker.com/docker-for-mac/>`_
+ - Install Docker for `Windows  <https://docs.docker.com/engine/installation/linux/>`_
```
Comment (aloupfor) [inline en/installation_guide/install_with_docker_for_contributors.rst]: It looks like you made a small mistake. OS names doesn't match the good OS installation links.

Also you may have to get rid of the leading white space on line 20-21-22.

File: en/installation_guide/install_with_docker_for_contributors.rst [hunk 2]
```diff
@@ -11,176 +11,94 @@ Open Orchestra is composed of three projects:
...
 
 Install OpenOrchestra
 ---------------------
 
-First of all, you have to clone the Open Orchestra Docker repository:
+Clone the Open Orchestra applications repository and Docker repository in the same folder:
 
   .. code-block:: bash
-
+    $ cd <your_project_folder>/
```
Comment (aloupfor) [inline en/installation_guide/install_with_docker_for_contributors.rst:32]: I looked at the full file with the rst rendering and it looks like this code-block isn't shown.
I think you have to keep an empty line beetween `.. code-block:: bash` and the first line of code (here : `$ cd <your_project_folder>/`).

So, empty line missing (?).

File: en/installation_guide/install_with_docker_for_contributors.rst [hunk 3]
```diff
@@ -11,176 +11,94 @@ Open Orchestra is composed of three projects:
...
+    $ git clone git@github.com:open-orchestra/open-orchestra-media-demo.git
+    $ mkdir uploaded-files
 
-To contribute to the project a script can install two versions: the master version and the last stable one. The master branch is the branch which should be used to add new features to the project.
-Here is the directory tree with your two projects:
+Build Docker containers:
 
-.. code-block:: none
+  .. code-block:: bash
+    $ cd <your_project_folder>/open-orchestra-provision-docker/
```
Comment (aloupfor) [inline en/installation_guide/install_with_docker_for_contributors.rst:43]: empty line missing (?).

File: en/installation_guide/install_with_docker_for_contributors.rst [hunk 4]
```diff
@@ -11,176 +11,94 @@ Open Orchestra is composed of three projects:
...
-    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-front-demo ./open-orchestra-front-demo -s dev --ignore-platform-reqs --no-scripts --keep-vcs dev-master
-    $ [path-to-composer]/composer.phar create-project open-orchestra/open-orchestra-media-demo ./open-orchestra-media-demo -s dev --ignore-platform-reqs --no-scripts --keep-vcs dev-master
+    open_orchestra_cms.mongodb.host: oo_mongo
+    fos_http_cache.proxy_client.varnish.servers: [oo_varnish:6081]
+    host_elastica: oo_elasticsearch
+    media.directory: /var/www/uploaded-files
 
-  Build Docker containers :
+  .. code-block:: bash
+    $ ./bin/grunt  #install assets
```
Comment (aloupfor) [inline en/installation_guide/install_with_docker_for_contributors.rst:68]: empty line missing (?).

**Comment by aloupfor** [comment]
I just made some remarks regarding the rst formatting.
I think there is still some unexpected render linked to the indentation (some blocks look like quotation but I don't think that they are).

**Comment by veolys-nanne** [comment]
:+1: after having explain clearer how to set default values for docker

File: en/installation_guide/install_with_docker_for_contributors.rst [hunk 5]
```diff
@@ -11,176 +11,97 @@ Open Orchestra is composed of three projects:
...
-
-    $ php -r "readfile('https://getcomposer.org/installer');" | php
-
-see `Download Composer`_
 
 Install OpenOrchestra
 ---------------------
 
-First of all, you have to clone the Open Orchestra Docker repository:
+Clone the Open Orchestra applications repository and Docker repository in the same folder:
```
Comment (ngilain) [inline en/installation_guide/install_with_docker_for_contributors.rst]: repositories

**Pull request merged**